Gems/ROS2/Code/Tests/UrdfParserTest.cpp

```diff
@@ -440,4 +447,16 @@ namespace UnitTest
         EXPECT_EQ(result, "/home/foo/ros_ws/install/foo_robot/meshes/bar.dae");
     }
 
+    TEST_F(UrdfParserTest, XacroParseArgs)
```

Comment:
Really awesome to see the tests! If there's time to check any other cases (maybe some incorrect data as input) to see how the code handles it, that would be great, but it's always a balance with how far you want to go